lib/WordpressRepository.js

```diff
@@ -0,0 +1,103 @@
+const {convert} = require("url-slug");
+
+class WordpressRepository {
+    constructor(wordpressClient) {
+        this.wordpressClient = wordpressClient
+    }
+    async fetchNews() {
+        let response = await this.wordpressClient.get('/wp-json/wp/v2/posts/?per_page=100&_embed&_fields=_links.author,_links.wp:featuredmedia,_embedded,title,content.rendered,date,categories&categories=587')
```

Comment:
I think the categories url was configurable via env?

Reply:
was just copy&pasta, https://github.com/FAForever/website/pull/487/files#diff-7e4783341040a4ba6619cd0157ccb8d361c450d4ac84148a2fe589414db924ddL22

maybe we can optimize it later? there are for sure other problems, like "options/fields" not used...